helpers/runtime.rb

```diff
@@ -9,4 +9,23 @@ def get_runtime_jwt_payload
     rescue JWT::DecodeError
     end
   end
+
+  def get_scope_from_github(runner, run_id)
+    return unless run_id
+
+    Clog.emit("Get runner scope from GitHub API") { {get_runner_scope: {ubid: runner.ubid, repository_ubid: runner.repository.ubid}} }
```

Comment:
run_id can be added here. You can also keep map with keys ubid, repository_ubid and run_id at the top and pass it to emit functions. While emitting the exception, you can merge the exception key.

Reply:
Makes sense. I changed it